src/libs/verification/alternatives.ts

```diff
@@ -0,0 +1,76 @@
+import { IElement } from '@types';
+import { OntoUMLType, ClassStereotype, RelationStereotype } from '@constants/.';
+import pluralize from 'pluralize';
+
+export enum VerificationAlternativeCode {
+  REPLACE_ONTOUML_CLASS_STEREOTYPE = 'REPLACE_ONTOUML_CLASS_STEREOTYPE',
+  ONTOUML_CLASS_NAME_TO_PLURAL = 'ONTOUML_CLASS_NAME_TO_PLURAL',
+  REMOVE_LITERALS = 'REMOVE_LITERALS',
+  REPLACE_ONTOUML_CLASS_STEREOTYPE_TO_ENUMERATION = 'REPLACE_ONTOUML_CLASS_STEREOTYPE_TO_ENUMERATION',
+  REMOVE_PROPERTIES = 'REMOVE_PROPERTIES',
+}
+
+/**
+ * Utility class for alternatives for the resolution of verification issues
+ *
+ * @author Claudenir Fonseca
+ * @author Lucas Bassetti
+ */
+export class VerificationAlternative {
+  code: string;
+  title: string;
+  description: string;
+  elements: string;
+
+  constructor(code: string, source: IElement, context?: IElement[]) {
+    this.code = code;
+    this.title = null;
+    this.description = null;
+    this.elements = null;
+
+    switch (code) {
```

Comment:
Consider replacing the switch for object literal. Check `VerificationIssue` comment